python/lsst/pipe/tasks/processCcdWithFakes.py

```diff
@@ -370,10 +385,13 @@ def copyCalibrationFields(self, calibCat, sourceCat):
                 missingFieldNames.append(fieldName)
         if missingFieldNames:
             raise RuntimeError(f"calibCat is missing fields {missingFieldNames} specified in "
-                               "calibrationFieldsToCopy")
+                               "fieldsToCopy")
 
-        self.calibSourceKey = calibSchemaMapper.addOutputField(afwTable.Field["Flag"]("calib_detected",
-                                                               "Source was detected as an icSource"))
+        if "calib_detected" not in calibSchemaMapper.getOutputSchema():
+            self.calibSourceKey = calibSchemaMapper.addOutputField(afwTable.Field["Flag"]("calib_detected",
+                                                                   "Source was detected as an icSource"))
```

Comment:
Is this the correct column doc still?

Reply:
This should only be added from the icSource cat as it will already exist the second time through.